i18n/README.fr.md

```diff
@@ -0,0 +1,413 @@
+
+[Résultats détaillés des tests d'Imagenette](https://docs.lightly.ai/self-supervised-learning/getting_started/benchmarks.html#imagenette)
+
+### CIFAR-10
+
+[Résultats détaillés des tests de référence CIFAR-10](https://docs.lightly.ai/self-supervised-learning/getting_started/benchmarks.html#cifar-10)
+
+## Terminologie
+Ci-dessous, vous pouvez voir un aperçu schématique des différents concepts du package.
+
+Les termes en gras sont expliqués plus en détail dans notre [documentation](https://docs.lightly.ai/self-supervised-learning/).
+
+<img src="/docs/source/getting_started/images/lightly_overview.png" alt="Présentation du package Lightly pip"/></a>
+
+### Prochaines étapes
+
+Rendez-vous sur la [documentation](https://docs.lightly.ai/self-supervised-learning/) et voyez les choses que vous pouvez réaliser avec Lightly !
+
+## Développement
+
+Pour installer des dépendances de développement (par exemple pour contribuer au framework) vous pouvez utiliser la commande suivante :
+
+```
+pip3 install -e ".[dev]"
+```
+
+Pour plus d'informations sur la façon de contribuer, jetez un œil [ici] (CONTRIBUTING.md).
+
+### Exécution de tests
+
+Les tests unitaires se trouvent dans le [répertoire tests](tests/) et nous vous recommandons de les exécuter en utilisant
+[pytest](https://docs.pytest.org/en/stable/). Il existe deux configurations de test
+disponible. Par défaut, seul un sous-ensemble sera exécuté :
+
+```
+faire des tests rapides
+```
+
+Pour exécuter tous les tests (y compris les plus lents), vous pouvez utiliser la commande suivante :
+
+```
+faire un test
+```
+
+Pour tester un fichier ou un répertoire spécifique, utilisez :
+
+```
+pytest <chemin d'accès au fichier ou au répertoire>
+```
+
+### Formatage du code
+
+Pour formater le code avec [black](https://black.readthedocs.io/en/stable/) et [isort](https://docs.pytest.org), exécutez :
+
+```
+créer un format
+```
+
+## Lectures complémentaires
+
+**Apprentissage auto-supervisé** :
+
+- Jetez un œil à notre [chaîne #papers sur Discord](https://discord.com/channels/752876370337726585/815153188487299083)
+  pour les derniers documents d'apprentissage auto-supervisés.
+- [Un livre de recettes d'apprentissage auto-supervisé, 2023](https://arxiv.org/abs/2304.12210)
+- [Les auto-encodeurs masqués sont des apprenants en vision évolutive, 2021](https://arxiv.org/abs/2111.06377)
+- [Propriétés émergentes des transformateurs de vision auto-supervisés, 2021](https://arxiv.org/abs/2104.14294)
+- [Apprentissage non supervisé des fonctionnalités visuelles par affectations de clusters contrastées, 2021](https://arxiv.org/abs/2006.09882)
+- [Ce qui ne devrait pas être contrastif dans l'apprentissage contrastif, 2020](https://arxiv.org/abs/2008.05659)
+- [Un cadre simple pour l'apprentissage contrastif des représentations visuelles, 2020](https://arxiv.org/abs/2002.05709)
+- [Momentum Contrast pour l'apprentissage de la représentation visuelle non supervisé, 2020](https://arxiv.org/abs/1911.05722)
```

Comment:
Do not translate paper names

Reply:
but then how can it be understood by them then

Reply:
We shouldn't modify original titles. The reader has to be familiar with english anyways as the work is written in english as well.

Reply:
This is still outstanding